distribution/docs/src/main/resources/_contents/configuring-from-the-command-console-contents.adoc

```diff
@@ -0,0 +1,164 @@
+.. Not required, but a good practice is to change the instance name (e.g., `federated_source`) of the file to something identifiable (`source1- ddf`).
+. Edit the copied file to etc with the settings for the configuration. (Refer to the table above to determine the configurable properties).
+.. This file is a Java properties file, hence the syntax is `<key>` = `<value>`.
+.. Consult the inline comments in the file for guidance on what to modify.
+.. The Configurable Properties tables in the Integrator's Guide for the Included Catalog Components also describe each field and its value.
+
+The new service can now be used as if it was created using the {admin-console}.
+
+===== Managing Applications From the {command-console}
+
+Applications can be installed from the {command-console} using the following commands:
+
+.App Commands
+[cols="2,5" options="header"]
+|===
+|Command
+|Effect
+
+|`app:add <appName>`
+|Install an app.
+
+|`app:list`
+|List all installed apps and current status.
+
+|`app:remove <appName>`
+|Uninstall an app.
+
+|`app:start`
+|Start an inactive app.
+
+|`app:status <appName>`
+|Detailed view of application status
+
+|`app:stop <appName>`
+|Stop an active app.
+
+|`app:tree`
+|Dependency tree view of all installed apps.
+|===
+
+===== Managing Features From the {command-console}
+
+. Determine which feature to install by viewing the available features on ${branding}. +
+`${branding-lowercase}${at-symbol}local>features:list`
+. The console outputs a list of all features available (installed and uninstalled). A snippet of the list output is shown below (the versions may differ):
+
+----
+State         Version          Name                          Repository  Description
+[installed  ] [2.0.1         ] ${ddf-branding-lowercase}-core                      ${ddf-branding-lowercase}-2.1.0
+[uninstalled] [2.0.1         ] ${ddf-branding-lowercase}-sts                       ${ddf-branding-lowercase}-2.1.0
+[installed  ] [2.0.1         ] ${ddf-branding-lowercase}-security-common           ${ddf-branding-lowercase}-2.1.0
+[installed  ] [2.0.1         ] ${ddf-branding-lowercase}-resource-impl             ${ddf-branding-lowercase}-2.1.0
+[uninstalled] [2.0.1         ] ${ddf-branding-lowercase}-source-dummy              ${ddf-branding-lowercase}-2.1.0
+----
+
+. Install the desired feature. +
+`${branding-lowercase}${at-symbol}local>features:install ${ddf-branding-lowercase}-source-dummy`
+. Check the feature list to verify the feature was installed. +
+`${branding-lowercase}${at-symbol}local>features:list`
+
+----
+State         Version          Name                          Repository  Description
+[installed  ] [2.0.1         ] ${ddf-branding-lowercase}-core                      ${ddf-branding-lowercase}-2.1.0
+[uninstalled] [2.0.1         ] ${ddf-branding-lowercase}-sts                       ${ddf-branding-lowercase}-2.1.0
+[installed  ] [2.0.1         ] ${ddf-branding-lowercase}-security-common           ${ddf-branding-lowercase}-2.1.0
```

Comment:
Looks like some of this is out of date now. Security common is no longer a bundle.